conf.py

```diff
@@ -374,5 +379,47 @@
                        "bus_device": ('https://circuitpython.readthedocs.io/projects/busdevice/en/latest/', None),
                        "register": ('https://circuitpython.readthedocs.io/projects/register/en/latest/', None)}
 
+# Adapted from sphinxcontrib-redirects
+from sphinx.builders import html as builders
+
+TEMPLATE = """<html>
+  <head><meta http-equiv="refresh" content="0; url=%s"/></head>
+</html>
+"""
+
+
+def generate_redirects(app):
+    path = os.path.join(app.srcdir, app.config.redirects_file)
+    if not os.path.exists(path):
+        app.info("Could not find redirects file at '%s'" % path)
+        return
+
+    # TODO(stephenfin): Add support for DirectoryHTMLBuilder
```

Comment:
Who is this?

Reply:
An author of sphinxcontrib-redirects.  We should probably remove the TODO here (in fact I thought I had -- I'll make a TODO about it :rofl:)